advanced/is-pattern-1/templates/is/identity-server-volume-claims.yaml

```diff
@@ -23,17 +23,11 @@ spec:
   volumeMode: Filesystem
   resources:
     requests:
-      storage: 2Gi
-  {{- if and (.Values.wso2.persistentRuntimeArtifacts.cloudProvider) (eq .Values.wso2.persistentRuntimeArtifacts.cloudProvider "internal-nfs") }}
-  storageClassName: nfs
-  {{- else if and (.Values.wso2.persistentRuntimeArtifacts.cloudProvider) (eq .Values.wso2.persistentRuntimeArtifacts.cloudProvider "external-nfs") }}
-  storageClassName: ""
-  selector:
-    matchLabels:
-      purpose: wso2is-pattern-1-tenants
-  {{- end }}
+      storage: 500M
```

Comment:
Is this value for storage recommended by the IS team?

Reply:
@ThilinaManamgoda yes. It might need even lesser than that but I have provided this much for a single Persistent Volume, just in case.